lib/arke/strategy/copy.rb

```diff
@@ -4,37 +4,33 @@
 
 module Arke::Strategy
   class Copy
-
     def start
```

Comment:
3- Strategy will create orderbook instance - cannot really get how to handle it correctly, should I merge exchanges orderbook to strategies orderbook? And next we operate only with merged orderbook isntead of smt like  exchange.orderbook.orders_queue.pop(&process_orders)?

Reply:
Just the strategy in initialize should create the order books and register call back events on it by passing a block

In this block we will queue the message to actions to send to rubykube exchange.

Start simple and don't worry about how existing code works